.github/workflows/pr.yml

```diff
@@ -81,7 +85,14 @@ jobs:
           cache: "pnpm"
 
       - run: pnpm i
-      - run: pnpm exec turbo run --affected lint
+      - run: pnpm exec turbo run --affected lint -- --output-file eslint_report.json --format json
+        continue-on-error: true
```

Comment:
JFYI you'll need to expand the step above this to see all eslint logs.

See https://github.com/frontpagefyi/frontpage/actions/runs/20170334186/job/57904083142